test-cases/longevity/longevity-lwt-500G-3d.yaml

```diff
@@ -0,0 +1,33 @@
+# Test duration 15 hours temporary. Should be 3 days. Will be changed later
+test_duration: 900
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50" ]
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(lwt_update_one_column=1)' cl=QUORUM duration=180m -mode native cql3 -rate threads=30" ,
+             "cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(lwt_update_two_columns=1)' cl=QUORUM duration=180m -mode native cql3 -rate threads=30"
+            ]
+stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml ops'(select=1)' cl=SERIAL duration=180m -mode native cql3 -rate threads=20" ]
+
+n_db_nodes: 4
+n_loaders: 3
+n_monitor_nodes: 1
+round_robin: true
```

Comment:
With round_robin I think the 3rd loader won't do anything with this configuration

Reply:
Why?